api_v3/lib/types/KalturaBaseUser.php

```diff
@@ -241,10 +243,13 @@ protected function validateNames($coreObject, array $names)
 					continue;
 				}
 			}
-			
+
 			if(myKuserUtils::startsWithSpecialChar($this->$kalturaProperty))
 			{
-				throw new KalturaAPIException(KalturaErrors::INVALID_FIELD_VALUE, $kalturaProperty);
+				if (strlen($this->$kalturaProperty) != 1) {
+					throw new KalturaAPIException(KalturaErrors::INVALID_FIELD_VALUE, $kalturaProperty);
+				}
+				$this->$kalturaProperty = self::CHAR_APOSTROPHE . $this->$kalturaProperty;
```

Comment:
lets do this on all cases where the name starts with invalid char instead of throwing an error (without checking the length